.travis.yml

```diff
@@ -18,8 +17,8 @@ env:
 matrix:
   allow_failures:
     - php: hhvm
-
     - php: hhvm-nightly
+    - env: DB=pgsql db_dsn='postgres://postgres@127.0.0.1/cakephp_test'
```

Comment:
Oh, thats related to a bug in the schema generation in CakePHP. See [this commit](https://github.com/cakephp/migrations/commit/a57b25e5e3ac6d144441f4ebba0c0d4f80c46275) for more details.